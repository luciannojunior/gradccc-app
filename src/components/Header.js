import React from 'react';
import { View, StyleSheet, Text } from "react-native";

const Header = (props) => {
    const { textStyle, viewStyle } = styles;

    return(
        <View style={viewStyle}>
            <Text style={textStyle}>{props.headerText}</Text>
        </View>
    );
}

const styles = StyleSheet.create({
    viewStyle: {
        backgroundColor: '#069',
        justifyContent: 'center',
        alignItems: 'center',
        height: 80,
        elevation: 1,
        position: 'relative',
        
    },
    textStyle: {
        fontSize: 20,
        color: '#fff'
    }
})

export default Header;