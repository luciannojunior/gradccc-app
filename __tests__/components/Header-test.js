import 'react-native';
import React from 'react';
import Header from '../../src/components/Header';

import renderer from 'react-test-renderer';

it('Header renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});