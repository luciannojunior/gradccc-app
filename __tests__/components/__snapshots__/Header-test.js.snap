// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#069",
      "elevation": 1,
      "flexDirection": "column",
      "height": 60,
      "position": "relative",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#fff",
        "fontSize": 20,
        "marginTop": 12,
      }
    }
  />
</View>
`;
